### PR TITLE
Add recipe for CoolProp

### PR DESCRIPTION
**Pull request by danilomendesdias**
Add recipe for CoolProp

### CODE REVIEW

File: recipes/CoolProp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```
Comment (synapticarbors) [inline recipes/CoolProp/meta.yaml]: Is `cython` needed?

Comment (danilomendesdias) [reply recipes/CoolProp/meta.yaml]: I added a thread on their mailing-list:
https://groups.google.com/forum/#!topic/coolprop-users/oKDLMXn14RI

I'll wait for a reply to correctly answer you.

Thank you @synapticarbors

Comment (danilomendesdias) [reply recipes/CoolProp/meta.yaml]: I investigated this by myself and I realized that it uses cython if possible, but it is not required.
So, answering your question: no, it does not depend on cython.

**Comment by danilomendesdias** [comment]
Is there anything else I can do?

File: recipes/CoolProp/meta.yaml (hunk 1 above)
Comment (danilomendesdias) [reply recipes/CoolProp/meta.yaml]: I think this piece of code answer you question:

https://github.com/CoolProp/CoolProp/blob/master/wrappers/Python/setup.py#L193

Besides that, c++ files which was auto-generated by Cython are in the tar.gz file from pypi.

**Comment by synapticarbors** [comment]
@danilomendesdias -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**